test/Makefile.am

```diff
@@ -77,7 +77,8 @@ TEST_CASES = \
 	test-0076.sh \
 	test-0077.sh \
 	test-0100.sh \
-	test-0101.sh
+	test-0101.sh \
```

Comment:
To be honest, I do not know why the list of tests is sparse ... but please make the new test-cases `test-0078` and `test-0079`.